spec/features/search_across_spec.rb

```diff
@@ -24,12 +24,18 @@
     tag1
   end
 
+  def js_search
+    within(first('.search-query-form')) do
+      find('#site-search-type').click
```

Comment:
What is the purpose of these steps?  Can the be navigated in an accessible way (e.g. Found with text)?

Reply:
Removing the old site partial means by default the view looks like this on load:  ![Screenshot 2024-09-25 at 3 47 20 PM](https://github.com/user-attachments/assets/fecdaf61-25de-4107-81b9-fd7a2008bf7e). There is no search button, so we need to click on the dropdown and choose ![Screenshot 2024-09-25 at 3 47 29 PM](https://github.com/user-attachments/assets/28f31579-0b1a-4a14-8860-94bc01bd6bda) which does have a search button.

Reply:
Can you do `click_on "Find exhibits by title"` and then `click_on "items in all exhibits"`?  That way we're also testing that this is accessible.

Reply:
done.